paasta_tools/cli/cmds/logs.py

```diff
@@ -100,9 +103,37 @@ def add_subparser(subparsers):
         default=DEFAULT_SOA_DIR,
         help="define a different soa config directory",
     )
+
+    status_parser.add_argument(
+        '-a', '--from', '--after', dest='time_from',
+        help='The ISO-8601 timestamp to get the logs from. Alternatively, for convenience, time differences can be '
+             'specified in a human readable way, such as --from 5m will grab logs from 5 minutes ago',
+    )
+    status_parser.add_argument(
+        '-t', '--to', dest='time_to',
+        help='The ISO-8601 timestamp to get logs up to. Follows same convention as --from for human readable '
+             'differences. Defaults to right now.'
+    )
+    status_parser.add_argument(
+        '-l', '--lines', dest='line_count',
+        help='The number of lines to retrieve from the specified offset. May optionally be prefixed with a "+" or "-" '
+             'to specify which direction from the offset, defaults to "-100"',
+        type=int,
+    )
+    status_parser.add_argument(
+        '-o', '--line-offset', dest='line_offset',
+        help='The offset at which line to start grabbing logs from. For example 1 would be the first line. Paired with '
+             '--count +100 would give you the first 100 lines of logs. Defaults to the latest line\'s offset',
```

Comment:
Kinda, I don't really like that particular part about tail's syntax. I much prefer this way of doing it where you specify which offset and then the line count in either direction.